src/Requests/Companies/PostRequest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace TruckersMP\Requests\Companies;
+
+use TruckersMP\Models\CompanyPost;
+use TruckersMP\Requests\Request;
+
+class PostRequest extends Request
+{
+    /**
+     * The ID of the requested company.
+     *
+     * @var int
+     */
+    protected $companyId;
+
+    /**
+     * The ID of the requested post.
+     *
+     * @var int
+     */
+    protected $postId;
+
+    /**
+     * Create a new PostRequest instance.
+     *
+     * @param int $companyId
+     * @param $postId
+     */
+    public function __construct(int $companyId, $postId)
```

Comment:
`$postId` should be probably `int`, too. Also, do not forget to update the documentation.